android/specs/local_test.js

```diff
@@ -0,0 +1,44 @@
+const { Builder, By, until } = require("selenium-webdriver");
+var assert = require('assert');
+
+var buildDriver = function() {
+  return new Builder()
+    .usingServer('http://127.0.0.1:4723/wd/hub')
+    .build();
+};
+
+describe('BrowserStack Local Testing', function() {
+  this.timeout(0);
+
+  it('check tunnel is working', async function () {
+    let driver = buildDriver();
+    try {
+        await driver.wait(
+          until.elementLocated(
+            By.xpath(
+              '/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout[2]/android.view.ViewGroup/android.widget.Button'
```

Comment:
Is there a better way than xpath for this? Can use Appium Inspector to find an attribute

Reply:
- We have used plain `selenium` library to write the script. many functionality related to appium would not have worked here.
- I tried using existing sample repos locators but that didn't worked.
- Didn't put much effort and reffered to existing nodejs sdk sample repo for these locators.
- I believe these scripts for sample repo should act fine because this is a sample script and user won't be looking this script as a reference.